ee/app/livechat-enterprise/server/lib/QueueInactivityMonitor.js

```diff
@@ -0,0 +1,89 @@
+import { SyncedCron } from 'meteor/littledata:synced-cron';
+import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
+import moment from 'moment';
+
+import { settings } from '../../../../../app/settings/server';
+import { Logger } from '../../../../../app/logger';
+import { LivechatRooms, Users, LivechatInquiry } from '../../../../../app/models/server';
+import { Livechat } from '../../../../../app/livechat/server/lib/Livechat';
+
+export class QueueInactivityMonitor {
```

Comment:
Have you talked about using `Agenda` package? 
We introduced something similar [here](https://github.com/RocketChat/Rocket.Chat/blob/develop/ee/app/livechat-enterprise/server/lib/AutoCloseOnHoldScheduler.ts).
The solution you implemented works, I just want to make sure you can rely on other alternatives too(Perhaps with less complexity).